timereport/.chalice/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "2.0",
+  "app_name": "timereport",
+  "stages": {
+    "dev": {
+      "api_gateway_stage": "api",
+      "environment_variables": {
+                                "SLACK_TOKEN": "fake_token",
+                                "backend_url": "https://ywdi37qne9.execute-api.eu-north-1.amazonaws.com/api/timereport",
```

Comment:
Do we really want to check in the real endpoint like this?

Reply:
The config file could be added to gitignore.

Reply:
no because the config file is required if we want to "deploy" it using CI/CD

Reply:
The config file could be written during that process. Or we could leverage os env vars instead.
I don't like that we have a config file that can contain sensitive information within the repo

Reply:
Oh right I was thinking of our own config.json. This is for chalice.
But maybe chalice has a good way to specify config without checking in a plain text file?

Reply:
But if you feel comfortable to have it like it is now you can keep it if you want.